webapp/src/Controller/BaseController.php

```diff
@@ -255,6 +262,10 @@ protected function deleteEntity(
                 $entityId = $entity->getTeamid();
             }
 
+            // Get the contests to trigger the event for. We do this before
+            // deleting the entity, since linked data might have vanished
+            $contestsForEvents = $this->getEventContestsForEntity($entity, $DOMJudgeService);
```

Comment:
I think the name of this function is not very clear. Also the variable and function names don't match.
Maybe rename the method to `getContestsForEventsForEntity` although I'm not a big fan of such long names.

Reply:
I dont like such a long name either.... But I agree it's not clear.

Reply:
Gave it a shot at a better name